playground/backend/internal/setuper/setuper_test.go

```diff
@@ -0,0 +1,299 @@
+package setuper
+
+import (
+	playground "beam.apache.org/playground/backend/internal/api/v1"
+	"beam.apache.org/playground/backend/internal/environment"
+	"beam.apache.org/playground/backend/internal/executors"
+	"beam.apache.org/playground/backend/internal/fs_tool"
+	"beam.apache.org/playground/backend/internal/validators"
+	"context"
+	"github.com/google/uuid"
+	"os"
+	"reflect"
+	"testing"
+)
+
+func TestSetupCompileBuilder(t *testing.T) {
+	pipelineId := uuid.New()
+	sdk := playground.Sdk_SDK_JAVA
+	lc, err := fs_tool.NewLifeCycle(sdk, pipelineId, "")
+	if err != nil {
+		panic(err)
+	}
+	executorConfig := &environment.ExecutorConfig{
+		CompileCmd:  "MOCK_COMPILE_CMD",
+		RunCmd:      "MOCK_RUN_CMD",
+		CompileArgs: []string{"MOCK_COMPILE_ARG"},
+		RunArgs:     []string{"MOCK_RUN_ARG"},
+	}
+	val := validators.GetJavaValidators(lc.GetAbsoluteExecutableFilePath())
+
+	type args struct {
+		lc             *fs_tool.LifeCycle
+		sdk            playground.Sdk
+		executorConfig *environment.ExecutorConfig
+	}
+	tests := []struct {
+		name string
+		args args
+		want *executors.CompileBuilder
+	}{
+		{
+			name: "all success",
```

Comment:
What success? All? All in the whole world?

Reply:
Renamed tests.